BedrockServer.h

```diff
@@ -470,4 +470,7 @@ class BedrockServer : public SQLiteServer {
     // This is just here to allow `poll` in main.cpp to get interrupted when the server shuts down.
     // to wait up to a full second for them.
     SSynchronizedQueue<bool> _notifyDone;
+
+    atomic<size_t> _maxSocketThreads{500};
```

Comment:
Can we make this higher, like a middle ground between this and the previous 25k one?

Reply:
There was no previous maximum here. My reasoning with this is that things are already looking broken when this number is in the low 1000s. We could make it 1000?